spec/controllers/information_controller_spec.rb

```diff
@@ -2,6 +2,10 @@
 require 'rails_helper'
 
 RSpec.describe InformationController, type: :controller do
+  before do
+    user = FactoryGirl.create(:user) # add here
```

Comment:
So you do:

describe .... do
  user
  user.save
  etc...

And then the tests